src/bundle/Resources/translations/headline.en.xliff

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
will we add `ibexa` prefix to the domain? The current looks very generic to me

Reply:
Yes we should prefix our translations with `ibexa_` to avoid name collisions when someone wants to translate it on project level.

Reply:
but it would be a first prefixed domain in admin-ui, do we want to break the consistency?